Doc/reference/import.rst

```diff
@@ -982,9 +982,9 @@ __main__.__spec__
 Depending on how :mod:`__main__` is initialized, ``__main__.__spec__``
 gets set appropriately or to ``None``.
 
-When Python is started with the :option:`-m` option, ``__spec__`` is set
-to the module spec of the corresponding module or package. ``__spec__`` is
-also populated when the ``__main__`` module is loaded as part of executing a
+When Python is started with the :option:`-m` option, ``__main__.__spec__`` is set
+to the module spec of the corresponding module or package. ``__main__.__spec__`` is
+also populated when the :mod:`__main__` module is loaded as part of executing a
```

Comment:
Code markup is used on purpose to avoid creating 50 links when a module or function is named 50 times in a paragraph, that’s why there is `:mod:__main__` on line 982 but just `__main__` here.

Reply:
@merwok Instead of manual markup, can pydoc be taught to create a link for the first occurence in a paragraph only? It will allow to not bother about visual pleasure and focus on semantics (a normal word vs a term).